src/artifact_providers/__tests__/github.test.ts

```diff
@@ -1,13 +1,21 @@
 jest.mock('../../utils/githubApi.ts');
-import { getGitHubClient} from '../../utils/githubApi';
-import { GitHubArtifactProvider, ArtifactItem, lazyRequest, lazyRequestCallback} from '../github';
+import { getGitHubClient } from '../../utils/githubApi';
+import {
+  GitHubArtifactProvider,
+  ArtifactItem,
+  lazyRequest,
+  lazyRequestCallback,
+} from '../github';
```

Comment:
the reformat makes it really difficult to see what changed here -- can you discard those changes?

can probably set up formatting separately